aten/src/ATen/native/cuda/UnaryGammaKernels.cu

```diff
@@ -49,18 +49,53 @@ void trigamma_kernel_cuda(TensorIteratorBase& iter) {
   #endif // USE_JITERATOR
 }
 
-// TODO: jiterate the polygamma kernel (requires supporting heterogenous functor args)
+// Note [polygamma jiterator]
```

Comment:
Update note name

Reply:
Maybe merge this with the main jiterator note?